### PR TITLE
jsoncpp: new package at 1.9.2

### PR DESCRIPTION
**Pull request by ghost**
Another package needed for minetest.

### CODE REVIEW

File: jsoncpp/build [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        -DJSONCPP_WITH_CMAKE_PACKAGE=OFF \
+        -DJSONCPP_WITH_PKGCONFIG_SUPPORT=OFF \
+        "$@"
+
+    cmake --build   build
+    cmake --install build
+}
+
+mk -DBUILD_SHARED_LIBS=OFF
+mk -DBUILD_SHARED_LIBS=ON
```
Comment (dylanaraps) [inline jsoncpp/build]: Just build the shared library. :+1:

**Comment by dylanaraps** [comment]
Also, games are better suited to a user repository. See https://github.com/sdsddsd1/kiss-games

**Comment by sdsddsd1** [comment]
I wait merging this until minetest is ready.

**Comment by ghost** [comment]
I'll submit another pull request later, I have to recreate my fork.

**Pull request closed without merging**